charts/brig/values.yaml

```
@@ -25,7 +25,9 @@ config:
     region: "eu-west-1"
     sesEndpoint: https://email.eu-west-1.amazonaws.com
     sqsEndpoint: https://sqs.eu-west-1.amazonaws.com
-    dynamoDBEndpoint: https://dynamodb.eu-west-1.amazonaws.com
+    dynamoDBEndpoint: # https://dynamodb.eu-west-1.amazonaws.com
```

Can you omit this value entirely?

```suggestion
    # dynamoDBEndpoint: https://dynamodb.eu-west-1.amazonaws.com
```